python/ray/dashboard/client/src/pages/dashboard/logical-view/Actor.tsx

```diff
@@ -242,6 +250,13 @@ class Actor extends React.Component<Props & WithStyles<typeof styles>, State> {
                 </React.Fragment>
               ))}
               ){" "}
+              {actor.state === 0 ? (
+                <span className={classes.action} onClick={this.killActor}>
+                  Kill Actor
+                </span>
+              ) : (
+                ""
```

Comment:
You can use conditional rendering to avoid this trivial case, e.g. `actor.state === 0 && <span>...</span>`.

Reply:
Oh! Thanks for letting me know, I forgot about that option. I will add that in a new PR along with the other changes.